app/form_models/managed_organisations_form.rb

```diff
@@ -4,11 +4,14 @@ class ManagedOrganisationsForm
 
   attr_accessor :managed_organisations, :managed_school_ids
 
+  attr_reader :sort_column
+
   validate :at_least_one_option_selected
 
-  def initialize(params = {})
+  def initialize(sort, params = {})
```

Comment:
I'd probably prefer a form just for sorting so as not to conflate the two things that really don't need to be related. But if we don't do that, then `sort: nil` as the second parameter would allow you to only specify the sort when it's relevant